src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/IVsTestConsoleWrapper.cs

```diff
@@ -4,6 +4,7 @@
 namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer.Interfaces
 {
     using System.Collections.Generic;
+    using System.Threading.Tasks;
```

Comment:
I ended up not having VsTestConsoleWrapper implement both interfaces, since it has a constructor that takes ITranslationLayerRequestSender as an injected dependency, and all the async methods would require ITranslationLayerRequestSenderAsync instead. Instead, I just forked a new class for the async interface.